### PR TITLE
Add LunaSea.app v6.0.0

### PR DESCRIPTION
**Pull request by thomas-rx**
Adding the Cask for the LunaSea application

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/lunasea.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "lunasea" do
+  version "6.0.0"
+  sha256 :no_check
```
Comment (neutric) [inline Casks/lunasea.rb:3]:
```
version "6.0.0,60000006"
sha256 "10ad53618b83ca1b94ef0190805911f54c73865fd477695191991c41223454d2"
```

File: Casks/lunasea.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  url "https://github.com/CometTools/LunaSea/releases/download/6.0.0%2B60000006/LunaSea-macOS-60000006.zip"
```
Comment (neutric) [inline Casks/lunasea.rb:5]:
```
url "https://github.com/CometTools/LunaSea/releases/download/#{version.csv.first}%2B#{version.csv.second}/LunaSea-macOS-#{version.csv.second}.zip"
```

File: Casks/lunasea.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  name "LunaSea"
+  desc "Is a fully featured, open source self-hosted controller"
+  homepage "https://github.com/CometTools/LunaSea"
+
```
Comment (neutric) [inline Casks/lunasea.rb:9]:
```
livecheck do
  url :url
  strategy :git do |tags|
    tags.map do |tag|
      match = tag.match(/^v?(\d+(?:\.\d+)+)[+-](\d+(?:\.\d+)*)$/i)
      next if match.blank?

      "#{match[1]},#{match[2]}"
    end
  end
end
```
References: [`duplicati.rb`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/duplicati.rb) and [`qdesktop.rb`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/qdesktop.rb).

**Review by neutric — COMMENTED** [review]
Thanks for your first contribution to `homebrew-cask`, @xrths ! My review is in the code. Take it with a grain of salt as I am not a maintainer.

**Comment by miccal** [comment]
Thank you @xrths.

**Pull request merged**